plugins/outputs/stackdriver/README.md

```diff
@@ -0,0 +1,18 @@
+# Stackdriver Output Plugin
+
+This plugin writes to the [Google Cloud Stackdriver API](https://cloud.google.com/monitoring/api/v3/)
+and requires [authentication](https://cloud.google.com/docs/authentication/getting-started) with Google Cloud using either a service account or user credentials.
+
+Requires `project` to specify where Stackdriver metrics will be delivered to.
+
+Metrics are grouped by the `namespace` variable and metric key - eg: `custom.googleapis.com/telegraf/system/load5`
+
```

Comment:
We should probably mention the costs and link to https://cloud.google.com/stackdriver/pricing